miner/worker.go

```diff
@@ -885,6 +1119,19 @@ func (w *worker) commitTransactions(txs *types.TransactionsByPriceAndNonce, coin
 			log.Trace("Skipping unsupported transaction type", "sender", from, "type", tx.Type())
 			txs.Pop()
 
+		case errors.Is(err, circuitcapacitychecker.ErrBlockRowUsageOverflow):
+			log.Trace("Circuit capacity limit reached in a block") // TODO: add more logs
+			break loop
```

Comment:
Why try again here?

Reply:
it's not "try again"
it will quit the for loop

Reply:
oh I see.